appbuilder/device-emitter.ts

```diff
@@ -66,8 +66,8 @@ class DeviceEmitter extends EventEmitter {
 	}
 
 	private attachApplicationChangedHandlers(device: Mobile.IDevice): void {
-		device.applicationManager.on("applicationInstalled", (applicationName: string, applicationSettings: {isLiveSyncEnabled: boolean}) => {
-			this.emit("applicationInstalled", device.deviceInfo.identifier, applicationName, applicationSettings);
+		device.applicationManager.on("applicationInstalled", (applicationName: string) => {
```

Comment:
I'd rather you renamed `applicationName` to `applicationIdentifier` so it's not as misleading